packages/backend/package.json

```diff
@@ -8,13 +8,16 @@
     "lint": "eslint ./src ../shared --ext .ts",
     "lint:fix": "eslint ./src ../shared --ext .ts --fix"
   },
+  "type": "module",
```

Comment:
I'm guessing your TS issues were linked to this ?

Reply:
different plugin but yes, this missing field made TS angry